app/pages/dashboard/getQueries.js

```diff
@@ -0,0 +1,162 @@
+import _ from 'lodash';
+import { utils as vizUtils } from '@tidepool/viz';
+import utils from '../../core/utils';
+
+const chartPrefs = {
+  basics: {
+    stats: {
+      excludeDaysWithoutBolus: false,
+    },
+    sections: {},
+    extentSize: 14,
+  },
+  daily: {
+    extentSize: 1,
+  },
+  trends: {
+    activeDays: {
+      monday: true,
+      tuesday: true,
+      wednesday: true,
+      thursday: true,
+      friday: true,
+      saturday: true,
+      sunday: true,
+    },
+    activeDomain: '2 weeks',
+    extentSize: 14,
+    // we track both showingCbg & showingSmbg as separate Booleans for now
+    // in case we decide to layer BGM & CGM data, as has been discussed/prototyped
+    showingCbg: true,
+    showingSmbg: false,
+    smbgGrouped: false,
+    smbgLines: false,
+    smbgRangeOverlay: true,
+
+    // Formerly in viz.trends redux store
+    cbgFlags: {
+      cbg100Enabled: true,
+      cbg80Enabled: true,
+      cbg50Enabled: true,
+      cbgMedianEnabled: true,
+    },
+    focusedCbgDateTrace: null,
+    focusedCbgSlice: null,
+    focusedCbgSliceKeys: null,
+    focusedSmbg: null,
+    focusedSmbgRangeAvg: null,
+    showingCbgDateTraces: false,
+    touched: false,
+    stats: {
+      excludeDaysWithoutBolus: false,
+    },
+  },
+  bgLog: {
+    bgSource: 'smbg',
+    extentSize: 14,
+  },
+  agpBGM: {
+    bgSource: 'smbg',
+  },
+  agpCGM: {
+    bgSource: 'cbg',
+  },
+  settings: {
+    touched: false,
+  },
+  excludedDevices: [],
+};
+
+const getQueries = (
```

Comment:
To create this function `getQueries`, I started at [this function call to generatePDF()](https://github.com/tidepool-org/blip/blob/WEB-3336-tide-drawer/app/pages/patientdata/patientdata.js#L1061) and worked backwards up the callback chain, which should return that `queries` argument. 

Questions to follow.